biothings/tests/biothings_pytest_plugin/setup.py

```diff
@@ -0,0 +1,32 @@
+import os
```

Comment:
You could probably consolidate this to only use a pyproject.toml file by combining the setup.py information